internal/cmd/public-ip/ip-ranges/ip-ranges.go

```diff
@@ -0,0 +1,26 @@
+package ipranges
```

Comment:
Please rename all occurences of "ip-ranges" into "ranges" (docs, directory, package and imports) 

Desired naming "stackit public-ip ranges list"